dbuf-core/src/ast/parsed/location.rs

```diff
@@ -47,3 +68,48 @@ where
         self.start + self.length
     }
 }
+
+impl<Pos: Copy> Span for Location<Pos>
```

Comment:
I think it would be simpler to use an already available `Span` implementation for `Range<Offset>`